cirro/cli/interactive/download_args.py

```diff
@@ -72,12 +82,24 @@ def ask_dataset_files_list(files: List[File]) -> List[File]:
         ]
     })
 
-    return [
+    selected_files = [
         file
         for file in files
         if file.relative_path in set(answers['files'])
     ]
 
+    if len(selected_files) == 0:
+        if prompt_wrapper({
```

Comment:
Could you use the ask function `ask("confirm", "No files were selected - try again?")`

Reply:
Much better (dcaf57d)